Phrases.java

```diff
@@ -1,232 +1,98 @@
-      "caressed his cheek",
-      "bluntly bit his nose",
-      "desparately punched his eyebrow",
-      "stroked his eye",
-      "made a quick punch in his throat",
-      "poked a finger in his mouth",
-      "punched his forehead with all the dope",
-      "punched his forehead with all the dope",
-      "slapped him from all nonsense",
-      "poked a finger in his eye"
-    },
-    {
-      "quickly hit in the pit of his stomach",
-      "made a double somersault and hit his chest",
-      "shamelessly pinched his nipples",
-      "cunningly poked a finger in his navel",
-      "made a thourough punch in his back",
-      "neatly made a punch in his heart"
-    },
-    {
-      "subtly made a hit in the legs",
-      "heartlessly broke his knee",
-      "grossly bit opponent's <censored>",
-      "gently caressed his toe",
-      "unthinkingly stepped on his foot",
-      "cowardly damaged his <censored>",
-      "rudely bit his ankle",
-      "shamelessly scratched his shin",
-      "jauntily poked a finger in his <censored>"
+  private static String readJsonFile(String filename) {
+    String jsonStr = "";
+    try {
+      FileReader fr = new FileReader(filename);
+      BufferedReader br = new BufferedReader(fr);
+      String line = "";
+      while ((line = br.readLine()) != null) {
+          jsonStr += line;
+      }
+    } catch (Exception e) {
+      System.out.println(e);
     }
-  };
 
-  static final String[] wasTrying = {
-    "was trying",
-    "attempted",
-    "made an effort",
-    "strived",
-    "decided",
-    "struggled",
-    "did his best"
-  };
+    return jsonStr;
+  }
 
-  static final String[][] toHit = {
-    {
-      "to punch the opponent in the face",
-      "to smash the opponent's face",
-      "to slam the enemy's nose with his heel",
-      "to caress opponent's cheek",
-      "to punch the competitor's eyebrow",
-      "to stroke the enemy's eye",
-      "to bite competitor's nose",
-      "to make a quick punch in the rival's throat",
-      "to poke a finger in the opponent's mouth",
-      "to punch the competitor's forehead",
-      "to slap the enemy",
-      "to poke a finger in the rival's eye"
-    },
-    { 
-      "to hit in the pit of the opponent's stomach",
-      "to hit the chest of the enemy",
-      "to pinch the opponent's nipples",
-      "to poke a finger in the navel of the enemy",
-      "to make a punch in the competitor's back"
-    },
-    {
-      "to make a hit in the opponent's legs",
-      "to brake the enemy's knee",
-      "to wound the rival's toe",
-      "to hit opponent's <censored>",
-      "to step on the competitor's foot",
-      "to damage the opponent's <censored>",
-      "to rudely bit the rival's ankle",
-      "to scratch the enemy's shin"
-    }
-  };
+  public static void initialize() {
+    JsonParser parser = new JsonParser();
+    Gson g = new Gson();
+    String lang = getLang();
 
+    String jsonStr;
+    try {
```

Comment:
It might be better to load the whole class - less code and will look cleaner. In this case json file is representation of the class properties. See Client.java for an example.

Reply:
sorry, I dont understand you. What means "load the whole class"?

Reply:
gson allows you not just to restore simple things like arrays of strings, but whole classes. It will go through each property of your class and assign the value from the file.  Example: https://github.com/vbo/fight_club_bot/blob/master/Storage.java#L48

So in your case you will have all your json files merged in one: `phrases.json` and then to restore the whole class you write something like this:
```
Phrases phrases = g.fromJson(jsonStr, Phrases.class);
```

You need to make sure that properties in the json file match exactly the properties defined in Phrases class.

====

gson позволяет тебе считывать не только простые структуры данных типо массивов и строк, он позволяет считывать целый класс. Gson пройдется по каждому свойству класса и приравняет им свойство из файла. Например: https://github.com/vbo/fight_club_bot/blob/master/Storage.java#L48

В твоем случае тебе нужно будет смерджить все json файлы в один `phrases.json` и затем чтобы восстановить целый класс ты пишешь что-то вроде:
```
Phrases phrases = g.fromJson(jsonStr, Phrases.class);
```

Нужно только убедиться, что свойства в json файле указаны также как свойства класса.